docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  app:
+    image: python:3.10-slim
+    volumes:
+      - ./src:/app
+    working_dir: /app
+    env_file:
+      - ./src/.env
+    command: sh -c "echo "Installing_Dependencies..." && pip install -r requirements.txt && pip install --upgrade pip && echo "Running_Bot..." && python -u ./bot.py"
```

Comment:
Yeah this is gross, since you only have 1 service you should be able to just build a docker container and have these like you've got in the docker file. Might have to still pass through the token as env vars or something. idk it works so fuck it

Reply:
make an issue for it plz

Reply:
idk im just pointing it out, ideally you'd want it as a docker container and not using compose. you decide if you want to consider changing it :)